utils/firestore.js

```diff
@@ -431,20 +433,21 @@ const removeParticipantsDataDestruction = async () => {
         for (const doc of currSnapshot.docs) {
             const batch = db.batch();
             const participant = doc.data();
-            const { isIsoDate } = require('./validation');
             const timeDiff = isIsoDate(participant[dateRequestedDataDestroyCId])
                 ? new Date().getTime() - new Date(participant[dateRequestedDataDestroyCId]).getTime()
                 : 0
 
             if (participant[destroyDataCategoricalCId] === requestedAndSignCId || timeDiff > millisecondsInTwoDays) {
+                let hasRemovedField = false;
                 const fieldKeys = Object.keys(participant)
                 const participantRef = doc.ref;
                 fieldKeys.forEach(key => {
-                    if (!stubRecordList.includes(key)) {
+                    if (!stubFieldArray.includes(key)) {
                         batch.update(participantRef, { [key]: admin.firestore.FieldValue.delete() });
+                        hasRemovedField = true
                     }
                 })
-                count++;
+                if (hasRemovedField) count++;
             }
             await batch.commit()
```

Comment:
An error message will be helpful when the batch commit fails.

Reply:
We already had it at `catch` statement
<img width="732" alt="image" src="https://github.com/episphere/connectFaas/assets/24559107/0657c9f3-f6c2-4404-8973-b273329df14a">